index.bs

```diff
@@ -211,8 +215,7 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
         In the context of this specification, a MIME type is also called content
         type. A <dfn>valid media MIME type</dfn> is a string that is a <a>valid
         MIME type</a> per [[mimesniff]] and must have only one parameter that is
-        named <code>codecs</code> which contains only one value that is not the
-        empty string.
+        named <code>codecs</code> with a value describing a simple media codec.
```

Comment:
You said "simple", you probably meant to say "single". Nothing simple about video codecs ;)